sci-physics/photos/photos-3.64.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit fortran-2 autotools
+
+MY_PN="PHOTOS"
+MY_P=${MY_PN}.${PV}
+
+DESCRIPTION="Monte Carlo for bremsstrahlung in the decay of particles and resonances"
+HOMEPAGE="
+	https://gitlab.cern.ch/photospp/photospp
+	http://photospp.web.cern.ch/photospp/
+"
+SRC_URI="https://photospp.web.cern.ch/resources/${MY_P}/${MY_P}.tar.gz"
+S="${WORKDIR}/${MY_PN}"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="doc examples hepmc pythia tauola"
+
+RDEPEND="
+	hepmc? ( sci-physics/hepmc:2=[-cm(-),gev(+)] )
+	sci-physics/hepmc:3=[-cm(-),gev(+)]
+	pythia? ( sci-physics/pythia:8= )
+	tauola? ( sci-physics/tauola[hepmc?,hepmc3,pythia?] )
+"
+DEPEND="${RDEPEND}
+	doc? (
+		app-text/doxygen[dot]
+		app-text/ghostscript-gpl
+		app-text/texlive
+	)
+"
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		--without-mc-tester \
+		--with-hepmc3="${EPREFIX}/usr" \
+		$(use_with hepmc hepmc "${EPREFIX}/usr") \
```

Comment:
Maybe we can align this with `evtgen.ebuild` where we have the `hepmc3` flag to choose hepmc:3 over hepmc2.

Reply:
sure, I tried to align them with what is used in the upstream configures.

Reply:
I think it is best to use `hepmc2` and `hepmc3` as flags to avoid confusions.